packages/lab/src/Flow/Flow.spec.ts

```diff
@@ -0,0 +1,367 @@
+import { test, expect, type Locator, type Page } from "@playwright/test";
+
+let flowCanvasLocator: Locator;
+let referencePage: Page;
+
+async function selectNode(
+  nodeGroup: string,
+  nodeName: string,
+  withExpand: boolean
+) {
+  const baseAssetNodeLocator = referencePage
+    .getByRole("listitem")
+    .filter({ hasText: nodeGroup });
+  if (withExpand) {
+    await baseAssetNodeLocator
+      .getByRole("button", { name: "Expand Group" })
+      .click();
+  }
+  return baseAssetNodeLocator.getByRole("button", { name: nodeName });
+}
+
+async function dragToPosition1(node: Locator) {
+  await node.dragTo(flowCanvasLocator, { targetPosition: { x: 120, y: 50 } });
+}
+
+async function dragToPosition2(node: Locator) {
+  await node.dragTo(flowCanvasLocator, { targetPosition: { x: 130, y: 400 } });
+}
+
+async function dragToPosition3(node: Locator) {
+  await node.dragTo(flowCanvasLocator, { targetPosition: { x: 620, y: 80 } });
+}
+
+async function connectNodes(
+  sourceNode: string,
+  sourceConnection: string,
+  destinyNode: string,
+  destinyConnection
```

Comment:
"Destiny" means else; did you mean "destination"?
Either way, I think we should use `target`, as it's what we're using in the Flow APIs

```suggestion
  targetNode: string,
  targetConnection: string,
```

Reply:
Changed in new commit.